Library/AppEnvironment.swift

```diff
@@ -240,7 +240,8 @@ public struct AppEnvironment {
           basicHTTPAuth: service.serverConfig.basicHTTPAuth
         ),
         oauthToken: service.oauthToken,
-        language: current.language.rawValue
+        language: current.language.rawValue,
+        currency: current.locale.currencyCode ?? "USD"
```

Comment:
We could do a more graceful regional fallback here...but not sure how to manage that.